### PR TITLE
docs: Add Pimcore quickstart, fixes #5843

### LINKED ISSUE #5843: Support Pimcore CMS
### Discussed in https://github.com/orgs/ddev/discussions/5840

<div type='discussions-op-text'>

<sup>Originally posted by **80982proton** February 14, 2024</sup>
Hello folks,

it would be great if you could add support for Pimcore. @dpfaffenbauer @brusch

Best regards</div>

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

* #5843 requests Pimcore support

We'll want more input from the Pimcore community, but this adds a basic quickstart.

Docs example at https://ddev--5909.org.readthedocs.build/en/5909/users/quickstart/#pimcore

### CODE REVIEW

**Comment by rfay** [comment]
@80982proton would love to have your review here.

File: docs/content/users/quickstart.md [hunk 1]
```diff
@@ -418,6 +418,19 @@ Visit the [Moodle Admin Quick Guide](https://docs.moodle.org/400/en/Admin_quick_
...
+
+Using the [Pimcore skeleton](https://github.com/pimcore/skeleton) repository:
+
+``` bash
+mkdir my-pimcore && cd my-pimcore
+ddev config --docroot=public
+ddev start
+ddev composer create pimcore/skeleton
+ddev exec pimcore-install --mysql-username=db --mysql-password=db --mysql-host-socket=db --mysql-database=db --admin-password=admin --admin-username=admin --no-interaction
+ddev launch
```
Comment (rpkoller) [inline docs/content/users/quickstart.md:431]:
```suggestion
ddev launch /admin
```
I would point directly to the login page that way the person testing is able to directly login with the previously set credentials and there is also no need to search for what the path to the admin login is. aside that the PR looks good

**Comment by SuperFlySly** [comment]
Looks good so far, there are still some fine details which you will have to install yourself! There is also the "symfony consumer command" that you have to start! https://github.com/pimcore/skeleton/blob/11.x/.docker/supervisord.conf


Bildschirm­foto 2024-02-28 um 20 14 31

**Comment by rfay** [comment]
@SuperFlySly could you please be explicit about 
> there are still some fine details which you will have to install yourself

Were you just referring to the "Recommended but not required" things in your screenshot? Should we be telling them how to do everything?

We can add the symfony consumer command with `web_extra_daemons` I guess (that installs in supervisord)

**Comment by rfay** [comment]
@SuperFlySly I've added the consumer as `web_extra_daemon`, would love your review again.

It looks to me like the other "recommended" things aren't something that would need to be done in a "quickstart", true?

**Comment by rfay** [comment]
Related: [Pimcore additional tools installation](https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Setup_and_Hosting/Additional_Tools_Installation/#pdf-generation)

**Comment by rfay** [comment]
I'm going to go with this, and we'll learn from the Pimcore community as they arrive! Thanks for the help everybody.

**Pull request merged**